cartography/intel/aws/util/common.py

```diff
@@ -5,7 +5,7 @@
 
 def parse_and_validate_aws_requested_syncs(aws_requested_syncs: str) -> List[str]:
     validated_resources: List[str] = []
-    for resource in aws_requested_syncs.split(','):
+    for resource in aws_requested_syncs:
```

Comment:
same as gcp, services would be a comma separated string like "iam,ec2,s3". Let's use the split function to split here.